Update Global Leads Directory with Lagos Co-Leads

emmaodia opened the pull request:
<!-- Dive straight into opportunities near you by joining a Developer Circle nearest to you :
👉  https://developers.facebook.com/developercircles/join/ -->

#### What does this PR do?
Added the name of the Co-Leads from Lagos to the Global Directory

emmaodia committed:
Update DevCGlobal Directory

- Added the names, github profiles of Co-Lead from Lagos, Nigeria.
- Added the link to the Facebook group of DevC Lagos.
- Added the link to the Github organization of DevC Lagos.

emmaodia committed:
Update DevCGlobalDirectory.md

emmaodia committed:
Update DevCGlobalDirectory.md

hervegithub commented:
Hello, your PR does not go through because you did not add a greeting in the comment of your PR

emmaodia commented:
Hi @hervegithub Okay. Thank you for pointing that out.
How do I fix this for some other time?
Thank you!

emmaodia committed:
Merge branch 'master' into master

emmaodia committed:
Added a New Article

Added an article on OAuth with PassportJS-Facebook using ExpressJS and MongoDB. This article teaches the reader how to implement Authentication and Authorizations in NodeJS Apps Using Facebook. It is published in the official DevC Lagos Medium Publication.

emmaodia commented on DevCGlobalDirectory.md:
```diff
@@ -374,8 +374,8 @@
     - [Emmanuel Odianosen Oaikhenan](https://github.com/innocentamadi)
     - [Innocent Amadi](https://github.com/emmaodia)
     - Peculiar Ediomo-Abasi
```
Thank you for highlighting. The Lead doesn't have a Github account.
